test/e2e/nsx_subnet_test.go

```diff
@@ -62,7 +62,7 @@ func TestDefaultSubnetSet(t *testing.T) {
 
 	portPath, _ := filepath.Abs("./manifest/testSubnet/subnetport_1.yaml")
 	err = applyYAML(portPath, E2ENamespace)
-	time.Sleep(10 * time.Second)
+	time.Sleep(30 * time.Second)
```

Comment:
why we change 10 * time.Second to to 30 * time.Second?

Reply:
In Subnet e2e, we'd better to make this longer to ensure that NSX subnet is created and subnet info is updated to SubnetSet CR status. Otherwise, we may easily hit the subnet e2e failure.